azuredevops/internal/service/core/data_teams.go

```diff
@@ -86,12 +86,21 @@ func dataTeamsRead(d *schema.ResourceData, m interface{}) error {
 			ToSlice(&projectIDList)
 	}
 
+	var top int
+	topData, topOk := d.GetOk("top")
+	if topOk {
+		top = topData.(int)
+	} else {
+		top = 2147483647
+	}
```

Comment:
Add `top` to the schema like `data_team` does:
```go
"top": {
				Type:         schema.TypeInt,
				Optional:     true,
				Default:      100,
				ValidateFunc: validation.IntAtLeast(1),
			},
```

```suggestion
	top := d.Get("top")
```